examples/js/lights/RectAreaLightUniformsLib.js

```diff
@@ -16,6 +16,61 @@ console.warn( "THREE.RectAreaLightUniformsLib: As part of the transition to ES6
 // by Eric Heitz, Jonathan Dupuy, Stephen Hill and David Neubelt
 // code: https://github.com/selfshadow/ltc_code/
 
+// toHalf(val) - Convert float32 array value to float16 stored in uint16 array value
+//
+// This function is duplicated from examples/jsm/loaders/RGBELoader.js
+// Source: http://gamedev.stackexchange.com/questions/17326/conversion-of-a-number-from-single-precision-floating-point-representation-to-a/17410#17410
+
+var floatView = new Float32Array( 1 );
+var int32View = new Int32Array( floatView.buffer );
+
+/* This method is faster than the OpenEXR implementation (very often
+* used, eg. in Ogre), with the additional benefit of rounding, inspired
+* by James Tursa?s half-precision code. */
+
+function toHalf( val ) {
```

Comment:
You have to put this function and both variables `floatView` and `int32View` into a closure. Otherwise they are defined in global scope.

Besides, the `examples/jsm` module is generate from the `examples/js` version via: `node utils/modularize.js`.

Reply:
I'm still getting the hang of JS closures, so hopefully this is in the correct style.

`jsm` version generated from `js`. Tested `jsm` on Chrome for Windows, iOS 13 and iOS 14.

Reply:
Looks good. Unfortunately, the closure is still required since global scripts don't have a module scope 😞 .